app/views/sample_types/_sample_attribute_form.html.erb

```diff
@@ -22,11 +22,14 @@
 <% allow_name_change = constraints.allow_name_change?(sample_attribute) %>
 <% seek_sample_multi = sample_attribute&.seek_sample_multi? %>
 <% link_to_self = sample_attribute && sample_attribute.deferred_link_to_self %>
-<% hide_seek_sample_multi = seek_sample_multi && displaying_single_page? && sample_attribute&.isa_tag.blank? && sample_attribute&.title.downcase.include?('input') %>
-<% isa_tags_options = IsaTag.all.map { |it| [it.title, it.id] } %>
+<% hide_seek_sample_multi = sample_attribute&.input_attribute? && displaying_single_page? %>
+<% level = sample_type.level %>
+<% isa_tags_options = IsaTag.allowed_isa_tags_for_level(level).map { |it| [it.title, it.id] } %>
 <% display_isa_tag ||= false %>
 <% isa_tag_id = sample_attribute&.isa_tag_id %>
 <% isa_tag_title = sample_attribute&.isa_tag&.title %>
+<% template_attribute_id = sample_attribute&.template_attribute_id %>
+<% allow_isa_tag_change = constraints.allow_isa_tag_change?(sample_attribute) %>
```

Comment:
aren't all these database queries and overheads only necessary if display_isa_tag is true ?